vita3k/modules/SceAppMgr/SceAppMgr.h

```diff
@@ -29,6 +29,21 @@ enum SceAppMgrErrorCode {
     SCE_APPMGR_ERROR_BGM_PORT_BUSY = 0x80803000 //!< BGM port was occupied and could not be secured
 };
 
+
+enum SceAppMgrEventType {
```

Comment:
Isn't SceAppMgrSystemEventType exactly this?

Reply:
is little diiferent, is @CreepNT have send me this enum and struct